helm/charts/determined/values.yaml

```diff
@@ -125,6 +128,20 @@ useNodePortForMaster: false
 #     username: determined
 #     password: password
 
+# saml (EE-only) enables Security Assertion Markup Language Integration, which is only available if
+# enterpriseEdition is true. It allows users to use single sign-on with their organization’s identity
+# provider.
+# saml:
+#   enabled:
+#   provider:
+#   idpRecipientUrl:
+#   idpSsoUrl:
+#   idpSsoDescriptorUrl:
+#   idpCertPath:
+#   autoProvisionUsers:
+#   groupsAttributeName:
```

Comment:
should be `groupsClaimName` to make it consistent

Reply:
i can put up a separate patch PR for that, as changing the variable names would probably require changing the docs & other files as well